internal/fs/inode/dir.go

```diff
@@ -639,20 +651,25 @@ func (d *dirInode) ReadDescendants(ctx context.Context, limit int) (map[Name]*Co
 			if o.Name == d.Name().GcsObjectName() {
 				continue
 			}
+			// Skip object with unsupported name (containing // or starting with /)
+			if storageutil.IsUnsupportedObjectName(o.Name) {
+				unsupportedObjects = append(unsupportedObjects, o.Name)
+				continue
+			}
 			name := NewDescendantName(d.Name(), o.Name)
 			descendants[name] = &Core{
 				Bucket:    d.Bucket(),
 				FullName:  name,
 				MinObject: storageutil.ConvertObjToMinObject(o),
 			}
 		}
+		logger.Debugf("Ignored unsupported objects: %v", unsupportedObjects)
```

Comment:
Switch all these debug logs to warning logs so that they show up even if log-level is set to info/warning.

Reply:
done